docs/contracts.rst

```diff
@@ -402,7 +402,7 @@ State variables can be declared as ``constant``. In this case, they have to be
 assigned from an expression which is a constant at compile time. Any expression
 that accesses storage, blockchain data (e.g. ``now``, ``this.balance`` or
 ``block.number``) or
-execution data (``msg.gas``) or make calls to external contracts are disallowed. Expressions
+execution data (``msg.value``) or make calls to external contracts are disallowed. Expressions
```

Comment:
I would still keep the gas example, since this is a little special. I actually only wondered today whether we allow `gasleft` in pure functions or not.

Reply:
Hm, perhaps it is best to provide both value and gas.